lib/elixir/lib/kernel.ex

```diff
@@ -1972,20 +1972,20 @@ defmodule Kernel do
   and the deletion will be considered a success.
   """
   @spec pop_in(Access.t, nonempty_list(term)) :: {term, Access.t}
-  def pop_in(data, keys)
-  def pop_in(nil, [h | _]), do: Access.pop(nil, h)
-  def pop_in(data, keys), do: do_pop_in(data, keys)
+  def pop_in(structure, keys)
```

Comment:
Do not rename `data` to `structure` please. We Typically use `data` or `term` or `value` for general entities in Elixir.